client/src/base-components/Drawer/__stories__/Drawer.stories.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import SearchIcon from '@mui/icons-material/Search';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import AccessibilityIcon from '@mui/icons-material/AccessibilityNew';
+import MenuIcon from '@mui/icons-material/Menu';
+
+import { MUI_PLACEMENTS } from '@utils';
+import Button from '../../Button/Button';
+import Drawer from '../Drawer';
+
+const actionHandler = action('onToggle');
+const handleEventAction = (event) => actionHandler(event);
+
+const itemsList = [
+  { name: 'Search', icon: <SearchIcon /> },
+  { name: 'Notifications', icon: <NotificationsIcon /> },
+  { name: 'Accessibility', icon: <AccessibilityIcon /> },
+  { name: 'Menu', icon: <MenuIcon /> },
+];
+
+export default {
+  title: 'base-components/Drawer',
+  parameters: {
+    controls: {
+      exclude: /^(onOpen|onClose|itemsList|itemsListStyle|childrenListStyle)$/g,
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          width: '90vw',
+          height: '90vh',
+          gap: '20px',
+          border: 'lightgrey 1px solid',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+  component: Drawer,
+};
+
+export const Default = () => <Drawer />;
+
+const Template = (args) => <Drawer itemsList={itemsList} {...args} />;
+
+export const Custom = Template.bind({});
+Custom.argTypes = {
+  anchor: {
+    control: 'inline-radio',
+    options: MUI_PLACEMENTS,
+    defaultValue: 'left',
+  },
+  open: {
+    control: { type: 'boolean' },
+    defaultValue: true,
+  },
+  variant: {
+    control: 'inline-radio',
+    options: ['temporary', 'permanent', 'persistent'],
+    defaultValue: 'persistent',
+  },
+};
+
+export const AnchorPlacements = () => {
+  const [activeDrawer, setActiveDrawer] = useState('');
+
+  return (
+    <>
+      {MUI_PLACEMENTS.map((placement) => (
+        <Button label={placement} onClick={() => setActiveDrawer(placement)} />
```

Comment:
you could use ButtonGroups on mui 
how over, in case of many buttons I expect to see them in row direction

Reply:
ok ill change this story